main.go

```diff
@@ -345,6 +350,7 @@ func sendSamples(logger log.Logger, w writer, timeseries []*prompb.TimeSeries) {
 		failedSamples.WithLabelValues(w.Name()).Add(float64(len(timeseries)))
 		writeErrors.Inc()
 	} else {
+		level.Debug(logger).Log("msg", "Sent samples", "num_samples", len(timeseries))
```

Comment:
This is already provided as a Prometheus metric, see line 355. What is the use case for having is as a log and a metric?

Reply:
This is fine because it is a debug log.